### PR TITLE
Bdd projects

### PR DESCRIPTION
**Pull request by Fx-Colliot**

### CODE REVIEW

**Review by Thoms-C — DISMISSED** [review]
Seem ok to me !

**Review by Marie-Garde — DISMISSED** [review]
Nice

File: index.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+const express = require("express");
+const app = express();
+const cors = require("cors");
+
+app.use(cors());
+app.use(express.json());
+
+app.get("/", (req, res) => {
+  res.send("Hi there!");
+});
```
Comment (LoicBrassart) [inline index.js]: Useless route

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please remove useless code

**Review by Thoms-C — APPROVED** [review]
Seem ok to me !

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**